client/src/components/StoreList.tsx

```diff
@@ -117,18 +123,73 @@ function StoreList() {
     }
   };
 
+  // Scan Store's QR code
+  const storeQR = async () => {
+    if (window.location == window.parent.location) {
+      const img = document.getElementById(
+        "sampleStoreQRImgSrc"
+      ) as HTMLImageElement;
+      processImageBarcode(img);
+    } else {
+      const imgReq = {
+        allowedMimeTypes: ["image/jpeg"],
+        allowedSources: ["camera"], // Restrict to camera scanning only
+      };
+      const imgRes = await window.microapps
+        .requestMedia(imgReq)
+        .then((res: any) => res);
```

Comment:
```.then(res: any)``` If you are using the result object as it is, then you dont need an additional  ```then```, it is essentially redundant.

Reply:
resolved in #22